recipe/meta.yaml

```diff
@@ -19,18 +19,18 @@ build:
         - vc14  # [win and py35]
 
 build:
-    number: 1
+    number: 2
 
 requirements:
     build:
         - python  # [win]
         - cmake  # [win]
         - zlib
-        - jpeg
+        - jpeg 8d
```

Comment:
Also, if there is a reason to use `jpeg 8d`, we should maybe just get rid of your `jpeg 9d` package and be done with all the version problems.